src/react/components/pages/appSettings/appSettingsPage.tsx

```diff
@@ -64,6 +64,9 @@ export default class AppSettingsPage extends React.Component<IAppSettingsProps>
                     <div className="my-3">
                         <p>{`${strings.appSettings.version.description} ${appInfo.version}`}</p>
                     </div>
+                    <div className="my-3">
+                        <p>{`${strings.appSettings.commit}: `} {process.env.REACT_APP_COMMIT_SHA}</p>
```

Comment:
this is Webpack build-time variable magic: https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables